app/models/folder.rb

```diff
@@ -10,26 +10,32 @@
 #  created_at  :datetime         not null
 #  updated_at  :datetime         not null
 #  team_id     :integer          default(0), not null
+#  personal_inbox :boolean       default(false)
 #
 
-#  Copyright (c) 2008-2017, Puzzle ITC GmbH. This file is part of
-#  Cryptopus and licensed under the Affero General Public License version 3 or later.
-#  See the COPYING file at the top-level directory or at
-#  https://github.com/puzzle/cryptopus.
-
 class Folder < ApplicationRecord
   belongs_to :team
-  has_many :encryptables, -> { order :name }, dependent: :destroy
+  has_many :encryptables, dependent: :destroy
 
   attr_readonly :team_id
 
   validates :name, presence: true
   validates :name, uniqueness: { scope: :team }
   validates :name, length: { maximum: 70 }
   validates :description, length: { maximum: 300 }
-
+  validate :only_one_personal_inbox
```

Comment:
may we use default arel functionality for this? https://apidock.com/rails/ActiveRecord/Validations/ClassMethods/validates_uniqueness_of

Reply:
Thanks for the info. 
Changed it to validates :personal_inbox, because rubocop is gränning:
![image](https://github.com/puzzle/cryptopus/assets/68543649/f390d698-cb19-4c46-b9e9-7464cb8331eb)

Reply:
I let the scope blank, because personal_inbox is a boolean attribute on folder model and this boolean only can be true on one folder